scripts/check_package_versions_in_docs.py

```diff
@@ -110,11 +110,25 @@ def get_public_id_from_yaml(configuration_file: Path):
 
     :param configuration_file: the path to the config yaml
     """
+
     data = unified_yaml_load(configuration_file)
-    author = data["author"]
+
+    # handle api spec files
+    if "openapi" in data:
+        return PublicId("author", "name", "0.0.0")
```

Comment:
We can do something better, we know there are only 5 valid config file names:
`aea-config.yaml`, `skill.yaml`,`connection.yaml`, `contract.yaml` and `protocol.yaml`. So let's check for inclusion and return otherwise. This should be done in the generator on line 126